docs/changehistory/NextVersion.md

```diff
@@ -168,11 +168,25 @@ Can also be attached as a map-layer:
 ### @itwin/core-common
 
 - [FontMap]($common) attempts to provide an in-memory cache mapping [FontId]($common)s to [Font](../learning/backend/Fonts.md) names. Use [IModelDb.fonts]($backend) instead.
+- Some types which are now more comprehensively exposed by backend's new ecschema-metadata integration were made deprecated:
```

Comment:
```suggestion
- Some types which are now more comprehensively exposed by backend's new `@itwin/ecschema-metadata` integration were made deprecated:
```

Maybe these deprecated methods should have suggested replacements right next to them?